### PR TITLE
Updates to PR #95

### PR DESCRIPTION
**Pull request by huangruizhe**
Updates to PR #95

### CODE REVIEW

**Comment by huangruizhe** [comment]
@danpovey Dan, this PR is ready to be merged when you think it is ok

File: scripts/check_dependencies.sh [hunk 1]
```diff
@@ -43,27 +43,22 @@ fi
 
 if which python >&/dev/null ; then
```
Comment (danpovey) [inline scripts/check_dependencies.sh]: I think we don't need for the default python to be python3 any more, as we are invoking it expliticly (but check via `git grep -w python | grep -v python3`).  So we could just check that python3 is installed here.

Comment (huangruizhe) [reply scripts/check_dependencies.sh]: Fixed.

**Comment by danpovey** [comment]
Thanks!
Can/did you check that it installs cleanly for you and that the examples run?

**Comment by huangruizhe** [comment]
Ok. I've installed a version of this PR from scratch, and run the `egs/swbd` and `egs/tedlium` recipe. It runs cleanly with no errors. For `egs/tedlium`, it took 5% more time to finish running (TOTAL time takes: 22663 => TOTAL time takes: 23836)

**Comment by danpovey** [comment]
Thanks!  Will merge.

**Pull request merged**